www/server/componentStatus/angular.json

```diff
@@ -0,0 +1 @@
+["alert-banner","alert-call","alert-meeting","alert","avatar","badge","button","button-group","call-control","checkbox","combo-box","composite-avatar","editable-textfield","form","icon","input","input-container","input-helper","input-message","input-section","label","lightbox","list","list-item","list-item-header","list-item-section","list-separator","loading","menu","modal","radio","slider","select","spinner","tabs","toggle-switch","topbar","accordion","sidebar","sidebar-body","sidebar-nav","sidebar-nav-item","sidebar-header","sidebar-footer","time-picker","date-picker","date-range-picker","breadcrumbs","popover","card","coachmark","page-header","tooltip","pagination","drawer","checkbox-group","radio-group","data-table","activity-button"]
```

Comment:
If these files are created by the build endpoint, they should not be tracked by git.